src/components/20-molecules/top-content-bar/index.scss

```diff
@@ -9,18 +95,20 @@
 
     min-height: 50px;
     vertical-align: middle;
-
-    text-align: center;
```

Comment:
do these changes impact the behaviour of the "existing" notification bar? That may be already in use , if our new designs collide with the existing implementation, this must be discussed.

Reply:
@Ben-Zahler 
If none of the top-content-bar attributes are set the only difference between new and old top-content-bar is the close functionality. The close icon is displayed and of course once you close it it will remain closed during that session.

<img width="726" alt="Screenshot 2021-12-28 at 11 21 27" src="https://user-images.githubusercontent.com/37440641/147556601-53f6b455-732e-42cd-86a4-921cdfbd4f0d.png">

If we should not display the close button for existing instances of top-content-bar then we need to know when to display it. In that case it will be hidden by default and we need some new attribute like "closable=true"
Ok like that or any other idea?

Reply:
that sounds all ver good, I think adding closeable=true is a good idea

Reply:
The attribute closeable added

Reply:
Top content bar used to be a super-simple component that displays a text and a button. Now it has the close icon, the left icon, both of them are optional and it also has different design for mobile devices. I had to change some of the existing code to merge the new functionality into the existing component. But the text is still center aligned, not with text-align:center, but with justify-content:center, but visually the text is centered as requested in the design. Also per the new design the component is now 64px high and also the font-size is 18px, Ben said that is agreed with the designer. All in all I had to change some of the old code to support the new design.